src/AppShed/Extensions/StorageBundle/Controller/TableController.php

```diff
@@ -0,0 +1,92 @@
+<?php
+/**
+ * Created by mcfedr on 05/05/2014 21:31
+ */
+
+namespace AppShed\Extensions\StorageBundle\Controller;
+
+use AppShed\Extensions\StorageBundle\Entity\Data;
+use AppShed\Extensions\StorageBundle\Entity\Store;
+use AppShed\Extensions\StorageBundle\Form\FiltersViewType;
+use AppShed\Remote\Element\Item\Link;
+use AppShed\Remote\Element\Item\Text;
+use AppShed\Remote\Element\Item\Thumb;
+use AppShed\Remote\Element\Screen\Screen;
+use AppShed\Remote\HTML\Remote;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\ParamConverter;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Method;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Template;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\Exception\AccessDeniedHttpException;
+use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
+
+class TableController extends StorageController
+{
+    /**
+     * @Route("/table/list", name="store_list")
+     * @Method({"GET"})
+     */
+    public function listAction(Request $request)
+    {
+        $app = $this->getApp($request);
+        $data['stores'] = $this->getDoctrine()->getRepository('AppShedExtensionsStorageBundle:Store')->findBy(['app' => $app]);
```

Comment:
Where did this $data appear from?

Maybe you can just pass a new array directly to `render`

    render(..., [
        'stores' => $this->doc...,
        'appParams' => $this->getEx...
    ])